bigbluebutton-html5/imports/api/presentations/server/methods/exportPresentationToChat.js

```diff
@@ -0,0 +1,56 @@
+import RedisPubSub from '/imports/startup/server/redis';
+import { check } from 'meteor/check';
+import { extractCredentials } from '/imports/api/common/server/helpers';
+import Logger from '/imports/startup/server/logger';
+import setPresentationExporting from '/imports/api/presentations/server/modifiers/setPresentationExporting';
+import Presentations from '/imports/api/presentations';
+
+const EXPORTING_THRESHOLD = 60000;
+
+export default function exportPresentationToChat(presentationId) {
+  const REDIS_CONFIG = Meteor.settings.private.redis;
+  const CHANNEL = REDIS_CONFIG.channels.toAkkaApps;
+  const EVENT_NAME = 'MakePresentationWithAnnotationDownloadReqMsg';
+
+  try {
+    const { meetingId, requesterUserId } = extractCredentials(this.userId);
+
+    check(meetingId, String);
+    check(requesterUserId, String);
+    check(presentationId, String);
+
+    const payload = {
+      presId: presentationId,
+      allPages: true,
+      pages: [],
```

Comment:
Just pointing out that there is already functionality supported here to pass arbitrary page numbers. Something to keep in mind for BBB 2.7 design iterations